pycqed/instrument_drivers/physical_instruments/QuTech_AWG_Module.py

```diff
@@ -1115,8 +1115,7 @@ def dio_calibration(cc, qwgs: List[QuTech_AWG_Module],
         if not cc:
             raise ValueError("Cannot calibrate QWGs; No CC provided")
 
-        if cc.ask("QUTech:RUN?") == '1':
-            cc.stop()
+        cc.stop()
```

Comment:
I believe the QWG driver was recently modified by @wvlothuizen and, therefore, we should not allow this PR to incorporate any conflicting changes!